.travis.yml

```diff
@@ -2,8 +2,8 @@
 language: python
 python: "2.7"
 before_install:
- - sudo apt-get update -qq
- - sudo apt-get install -qq python-apt python-pycurl
+  - sudo apt-get update -qq
+  - sudo apt-get install -qq python-apt python-pycurl
```

Comment:
Can you remove the formatting changes in formatting that are irrelevant to the Pull Request ?